x-pack/test/security_solution_cypress/cypress/e2e/detection_response/detection_alerts/alert_status.cy.ts

```diff
@@ -80,7 +79,7 @@ describe('Changing alert status', { tags: ['@ess', '@serverless'] }, () => {
         });
     });
 
-    it('can bulk open alerts', () => {
+    it.skip('can bulk open alerts', () => {
```

Comment:
Can we just create a ticket to track the skip and add here in a comment?

Reply:
Sure, adding to the code.

Ticket: https://github.com/elastic/kibana/issues/169091#issuecomment-1770502101

Reply:
Update: I skipped the whole suite as it started showing signs of flake again.

See [ticket.](https://github.com/elastic/kibana/issues/169091#issuecomment-1798553910)